README.rst

```diff
@@ -103,7 +103,9 @@ requirements file, just run ``tox``.
 The project also contains JavaScript tests which can be run using
 [Karma](karma-runner.github.io). ``tox`` will run the JavaScript tests
 after the Python tests. You can run only the JavaScript tests using
-``docker-compose run web tox -e js``.
+``docker-compose run web tox -e js``, or do continuous javascript
```

Comment:
We should have a consistent spelling for javascript, at least in the same paragraph. I'm unsure which is better, up to you